appengine/standard/firebase/firetactoe/firetactoe.py

```diff
@@ -94,25 +103,30 @@ def create_custom_token(uid, valid_minutes=60):
     security rules to prevent unauthorized access. In this case, the uid will
     be the channel id which is a combination of user_id and game_key
     """
-    header = base64.b64encode(json.dumps({'typ': 'JWT', 'alg': 'RS256'}))
```

Comment:
I kept the header above and separate from the payload intentionally, because it is unrelated and doesn't need to be thought of in conjunction with the payload. Moving it down below `client_email` interleaves the payload with the unrelated header, giving the impression that the header will be included in the payload, which it isn't.